### PR TITLE
PP-12004: Updates the refund statuses to account for ePDQ and Smartpay

### PR DESCRIPTION
**Pull request by NathanD-GDS**
https://payments-platform.atlassian.net/browse/PP-12004

### CODE REVIEW

**Comment by NathanD-GDS** [comment]
@oswaldquek Would you be able to review this PR? It's for a Jira ticket that you created just before Christmas!

File: source/refunding_payments/index.html.md.erb [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can see definitions, values, and the limits of every parameter and response
 |Status|Meaning|
 |---|---|
 |pending|You cannot refund the payment yet because your user has not completed the payment.|
-|unavailable|You cannot refund the payment, usually because the payment failed.|
+|unavailable|You cannot refund the payment. There are two possible reasons for this:<br><br><li>the payment failed<li>the payment was processed by a PSP that we no longer support, such as ePDQ or Smartpay|
```
Comment (alexbishop1) [inline source/refunding_payments/index.html.md.erb:55]: I think there’s actually more than two reasons. I think if a payment has already been fully refunded the status will be `unavailable`.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]

File: source/refunding_payments/index.html.md.erb (hunk 1 above)
Comment (NathanD-GDS) [reply source/refunding_payments/index.html.md.erb:55]: @alexbishop1 I've just tested it and a fully refunded payment returns `"status": "full"`

Comment (alexbishop1) [reply source/refunding_payments/index.html.md.erb:55]: Fair enough.

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**